### ISSUE TITLE
scheduler not really used(no "scheduler.step()")

### ISSUE THREAD

**Issue by gggc97** [mikgroup/subtle_data_crimes#2]
Hi,

in the training of deep learning model, e.g. in the file "Train_MoDL_Zpad.py", there is one line of code for a scheduler instance, but this scheduler is not used. Because there is no code for scheduler.step().

So is it designed in this way, or you forgot that line of code  "scheduler.step()"?

Best regards
Guang Chai
